grafana-plugin/src/containers/IncidentMatcher/IncidentMatcher.tsx

```diff
@@ -63,7 +63,7 @@ const IncidentMatcher = observer((props: IncidentMatcherProps) => {
       <div className={cx('columns')}>
         <div className={cx('incident-list')}>
           <Text.Title className={cx('title')} level={5}>
-            Matching Incidents
+            Matching Alert groups
```

Comment:
should the "g" in "groups" also be capitalized if we're going to capitalize "Alert"?